bandit/src/get-bandit-tests/get-bandit-tests.ts

```diff
@@ -1,3 +1,11 @@
+import * as AWS from "aws-sdk";
+import { queryChannelTests } from "./dynamo";
+
+const STAGE: string = process.env.Stage ?? "PROD";
+
+const docClient = new AWS.DynamoDB.DocumentClient({ region: "eu-west-1" });
+
 export async function run() {
```

Comment:
Let's make it clear what it's passing onto the next step:
`export async function run(): Promise<Test[]> {`